cmd/dfget/app/root.go

```diff
@@ -230,6 +230,8 @@ func initFlags() {
 		"identify whether supernode should skip secure verify when interact with the source.")
 	flagSet.IntVar(&cfg.ClientQueueSize, "clientqueue", config.DefaultClientQueueSize,
 		"specify the size of client queue which controls the number of pieces that can be processed simultaneously")
+	flagSet.BoolVar(&cfg.UploadMetrics, "uploadmetrics", false,
```

Comment:
How about making it true by default? 
I think we should encourage users could use the monitoring feature by default? WDYT？

Reply:
OK. I' ll change default to true